src/Propel/Generator/Behavior/AggregateColumn/AggregateColumnRelationBehavior.php

```diff
@@ -166,11 +237,17 @@ protected function addQueryUpdateRelated($builder)
         ]);
     }
 
+    /**
+     * @return \Propel\Generator\Model\Table|null
+     */
     protected function getForeignTable()
     {
         return $this->getTable()->getDatabase()->getTable($this->getParameter('foreign_table'));
     }
 
+    /**
+     * @return \Propel\Generator\Model\ForeignKey
```

Comment:
Is it possible to have 0 FKs returned?

Reply:
Unknown. Probably.

Reply:
Should we make it this?

```suggestion
     * @return \Propel\Generator\Model\ForeignKey|null
```

Reply:
If it is not intended to be that way than i would rather not.
In the future this could then throw exception maybe.

Reply:
PHPStan level 3-5 will tell us what to do here in the future anyway.